### PR TITLE
Refactor playlist capability handling slightly

### PR DESCRIPTION
**Pull request by theychx**
..so we can put additional code after the main clause in cli.cast.

### CODE REVIEW

File: catt/cli.py [hunk 1]
```diff
@@ -218,25 +217,22 @@ def cast(settings, video_url, subtitle, force_default, random_play, no_subs):
...
+            cst.play_playlist(stream.playlist_all_ids)
         else:
-            try:
-                if not stream.playlist_all_ids:
-                    raise ValueError
-                cst.play_playlist(stream.playlist_all_ids)
-                return
-            except (PlaybackError, ValueError):
+            if random_play:
+                stream.set_playlist_entry(random.randrange(0, stream.playlist_length))
```
Comment (skorokithakis) [inline catt/cli.py]: Do you want to split the call to `set_playlist_entry()` here outside the `if` so it's not repeated?

Comment (theychx) [reply catt/cli.py]: I'm not sure I understand what you mean.

Comment (skorokithakis) [reply catt/cli.py]: Just:

```python
if random_play:
    entry = random.randrange(0, stream.playlist_length)
else:
    warning("Playlist playback not possible, playing first video.")
    entry = 0
stream.set_playlist_entry(entry)
```

Comment (theychx) [reply catt/cli.py]: Ah, ok. Hang on.

Comment (theychx) [reply catt/cli.py]: I'm watching formula 1, so I'm not entirely focused. :smile:

Comment (skorokithakis) [reply catt/cli.py]: Haha, understandable :P

**Pull request merged**